components/brave_wallet/browser/json_rpc_service_unittest.cc

```diff
@@ -2046,6 +2046,104 @@ TEST_F(JsonRpcServiceUnitTest, GetERC721Balance) {
   EXPECT_TRUE(callback_called);
 }
 
+TEST_F(JsonRpcServiceUnitTest, GetERC1155Balance) {
+  // Invalid inputs.
+  bool callback_called = false;
```

Comment:
I think we should start using run_loop.Run + run_loop.Quit inside lambda from now on, please wrap GetERC1155TokenBalance to a test function.

Reply:
Updated